timm/models/resnet.py

```diff
@@ -341,7 +357,7 @@ def __init__(self, inplanes, planes, stride=1, downsample=None, cardinality=1, b
         self.conv3 = nn.Conv2d(width, outplanes, kernel_size=1, bias=False)
         self.bn3 = norm_layer(outplanes)
 
-        self.se = create_attn(attn_layer, outplanes)
+        self.se = create_attn(attn_layer, outplanes, **kwargs)
```

Comment:
These `**kwargs` get passed to Bottleneck using `block_args` in model_config as in L1112 model definition for `resnetrs152`